tests/run_container.sh

```diff
@@ -42,7 +42,7 @@ fi
 curl -s http://localhost:8080/pulp/api/v3/status/ | jq ".versions"
 
 # shellcheck disable=SC2064
-trap "${CONTAINER_RUNTIME} stop pulp" EXIT
+trap "${CONTAINER_RUNTIME} rm -f pulp" EXIT
```

Comment:
That makes sense. Another way to do that would be to run `docker logs pulp` in the exit handler and redirect to a file.

Reply:
I put some thought into this, and came to the following conclusions:
1. I like to have the container around (even locally) for post mortem analysis.
2. We could destroy the container before attempting to start a new one.
3. We could reuse an existing container instead of starting a new one every time (+ better startup time, - more side effects).
4. We should give the container a different name. "pulp" is to generic and taken from the oci-images example. People will get confused when this script starts messing around with it.
5. We use "the same (tm)" [as in copy'n'paste] in pulp-cli ci. So let's try not to add any squeezer specific features / names.

As i said before this script was never invented to be used outside of ephemeral ci, so a need for these changes seems reasonable when we try to make it fit for developer use.

Braindump for container names:
* pulp-test (probably still too generic)
* pulp-volatile
* pulp-ephemeral
* pulp-one-off
* pulp-tooling-ci

cc @ggainey you recently started to work extensively with local containers.